Learn_go/网络通信/learn02-web/json/main.go

```
@@ -0,0 +1,37 @@
+func main() {
+	testGetJson()
+}
+
+func testGetJson() {
+	path := "https://e.weather.com.cn/data/cityinfo/101010100.html"
+	r, _ := http.Get(path)
+	defer r.Body.Close()
+	b, _ := io.ReadAll(r.Body)
+	fmt.Printf("%v\n", string(b))
+	var jsonStr res
+	json.Unmarshal([]byte(b), &jsonStr)
```

golangci-lint (1.64.8)

Error return value of `json.Unmarshal` is not checked

(errcheck)